DesktopModules/Vanjaro/UXManager/Extensions/Menu/GoogleTagManager/Views/Setting/Setting.html

```diff
@@ -0,0 +1,22 @@
+﻿<div ng-controller="setting_setting" class="vj-ux-manager">
+    <uiengine provider="AngularBootstrap" identifier="setting_setting" class="uiengine-wrapper scrollbar">
+        <uilayout name="default">
+            <uielement name="ApplyTo" control="checkboxtoggle" label="false" attr-class="toggle_right btn_integration" container-attr-ng-show="ui.data.IsSuperUser.Options" ngmodel="ui.data.ApplyTo.Options"></uielement>
+        </uilayout>
+        <uilayout name="default" attr-ng-show="ui.data.ApplyTo.Options">
+            <uielement name="Host_Tag" control="textarea" attr-rows="20"></uielement>
+        </uilayout>
+        <uilayout name="default" attr-ng-show="!ui.data.ApplyTo.Options">
+            <uielement name="Site_Tag" control="textarea" attr-rows="20"></uielement>
```

Comment:
please add head tag and body tag text area control and remove tag text area control

Reply:
done the required changes, Please review